rest_framework_simplejwt/serializers.py

```diff
@@ -1,9 +1,9 @@
+from django.contrib.auth import get_user_model
```

Comment:
Combine the import statements:
`from django.contrib.auth import authenticate, get_user_model`